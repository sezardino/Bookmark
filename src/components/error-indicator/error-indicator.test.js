import React from 'react';
import {shallow} from 'enzyme';

import ErrorIndicator from './error-indicator';

describe('Component ErrorIndicator', () => {
	const component = shallow(<ErrorIndicator />);

	it('Component ErrorIndicator rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
