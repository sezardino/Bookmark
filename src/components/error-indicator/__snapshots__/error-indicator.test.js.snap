// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ErrorIndicator Component ErrorIndicator rendered correctly 1`] = `
"<div className=\\"popup popup--active\\">
  <div className=\\"popup__container\\">
    <div className=\\"popup__wrapper\\">
      <h2 className=\\"popup__title\\">
        Something goes wrong! reload browser
      </h2>
    </div>
  </div>
</div>"
`;
